examples/mcuboot/nsh/Kconfig

```diff
@@ -0,0 +1,12 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
+#
+
+config EXAMPLES_MCUBOOT_NSH
+	tristate "MCUboot NSH example"
+	default n
+	select BOOT_MCUBOOT
+	---help---
+		Example application to demonstrate a software firmware from
```

Comment:
> [...] demonstrate a software firmware

This is confusing, maybe some words are missing or misplaced here.
Also, the help text should describe **WHAT** the example is or intends to do, and less of the **HOW** part. This will improve the comprehension for the unaware user.

Reply:
```suggestion
		Example application to demonstrate the firmware image update and the update rollback processes of MCUboot using the swap upgrade method.
```
Just a suggestion.